Readme.md

```diff
@@ -27,9 +25,10 @@ g.load('/path/to/your/thesaurus')
 ```
 First, define the type URI for descriptors.
 If your thesaurus has a hierarchical structure that includes groups, 
+e.g., `skos:Collection`,
 you can optionally specify the type URI for sub-thesauri.
 In this case you should also specify the relation that relates sub-thesauri to concepts.
-Furthermore you can indicate whether this relation is a specialisation.
+Furthermore you can indicate whether this relation is a specialisation, in contrast to being a generalisation.
```

Comment:
Suggestion: " ... whether this relation is a specialisation relation (as opposed to a generalisiation relation, which is the default)."